### PR TITLE
chore: remove old libraries and minor ui fixes AP-308 AP-514

### PR DESCRIPTION
**Pull request by molimauro**
**What this PR does** 📖
Removes the `marked` library that is no more necessary and also replaces `VueMarkdown` when not needed.
When someone edits a message it should be parsed according to the new markdown rules (including spoilers, strikethrough, etc.).
Fixes some ui bugfix related to the chatbar (when you reply a message and then open also the commands-preview).

**Which issue(s) this PR fixes** 🔨
AP-514

**Special notes for reviewers** 🗒️

**Additional comments** 🎤
There is still a ui bug when you reply a message, open the commands-preview and open the files preview but this can be addressed only after a small refactor on the upload button.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,6 @@
     "less": "^4.1.1",
     "less-loader": "^7.1.1",
     "lodash": "^4.17.21",
-    "marked": "^4.0.7",
```
Comment (WanderingHogan) [inline package.json:65]: you love to see it, only a few more places and we can rip vue-markdown out, too

**Comment by WanderingHogan** [comment]
more subtractions than additions

image

**Review by WanderingHogan — APPROVED** [review]

**Comment by phillsatellite** [comment]
Tested: I noticed on Dev, when hovering over the emoji button in Edit Message the curser stays an arrow, in PR changes to a pointer finger is this expected?

https://user-images.githubusercontent.com/93608357/153318806-e90ce533-0b2d-463f-ac10-418e96845567.mov

File: package.json (hunk 1 above)
Comment (molimauro) [reply package.json:65]: yes exactly!

**Comment by molimauro** [comment]
> Tested: I noticed on Dev, when hovering over the emoji button in Edit Message the curser stays an arrow, in PR changes to a pointer finger is this expected?
> 
>  curser.mov

Yes i added the pointer when hovering that button because in the future we're going to display the emoji picker. That button is pretty useless rn, so if you prefer i can make any change :)

**Comment by stavares843** [comment]
just missing conflict 🔨

File: components/views/chat/message/Message.vue [hunk 2]
```diff
@@ -125,6 +129,7 @@ export default Vue.extend({
      * @method containsOnlyEmoji
      * @description Check wether or not a string only contains an emoji
      * @param str String to check against
+     * TO DO: is not working very well (:emoji: + "c")
```
[the remark this answers is not in the document]
Comment (molimauro) [reply components/views/chat/message/Message.vue:132]: thanks!

**Comment by molimauro** [comment]
done! 💪

**Pull request merged**